public/locales/fr-FR/translations.json

```diff
@@ -557,5 +557,7 @@
   "search_results_banner": null,
   "enable_amendment_name": null,
   "amendment_status": null,
-  "expected_date": null
+  "expected_date": null,
+  "transaction_type_name_AMMClawback": null,
```

Comment:
This should go up where the other transaction types are.

Reply:
Ditto with the others

Reply:
I put these at the bottom for other translators to find easily but sure I can put it back to the group